scripts/generate-local-token.js

```diff
@@ -0,0 +1,273 @@
+const { randomUUID } = require("crypto");
+const { KMSClient, SignCommand } = require("@aws-sdk/client-kms");
+
+// ============================================================================
+// simply launch this script with `node scripts/generate-local-token.js`
+// to generate a signed JWT token using local-kms
+// ============================================================================
+
+const LOCAL_KMS_ENDPOINT = "http://localhost:4566";
+const LOCAL_REGION = "eu-south-1";
+
+// KMS Key IDs from local-kms-seed/seed.yaml and jwks.json
+// Note: Only key2 is actually seeded in local-kms (see docker/local-kms-seed/seed.yaml)
+// The jwks.json contains both keys for verification purposes
+const LOCAL_KEY_IDS = {
+  // This key1 is in jwks.json but NOT in local-kms seed (cannot be used for signing)
+  key1: "41e3a8e9-5982-4a6d-b531-850774bfd961",
+  key2: "ffcc9b5b-4612-49b1-9374-9d203a3834f2",
+};
```

Comment:
Let's keep here only the local dev key we have in the local KMS (`ffcc9b5b-4612-49b1-9374-9d203a3834f2`), it's the only one we can actually use to sign a JWT locally -- the other keys we have in `jwks.json` are the public keys to verify dev env tokens, so that we can also accept locally tokens signed for the dev env, but we cannot create/sign tokens for those keys here